packages/canvas/canvas-graphics/src/CanvasGraphicsRenderer.ts

```diff
@@ -220,20 +228,30 @@ export class CanvasGraphicsRenderer
                 {
                     context.globalAlpha = fillStyle.alpha * worldAlpha;
                     context.fillStyle = contextFillStyle;
-                    context.fillRect(shape.x, shape.y, shape.width, shape.height);
+                    context.fillRect(
+                        (shape as Rectangle).x,
```

Comment:
I find all this casting to be a little too verbose. Why not creat a new const and cast it once and then use that property instead?

Reply:
I was a little tired this morning 😄 

I will update

Reply:
done